patientsearch/models/sync.py

```diff
@@ -194,10 +194,30 @@ def different(src, dest):
         return True
 
     if not different(src_patient, internal_patient):
-        return internal_patient
+        # If patient is active, proceed. If not, re-activate
+        if not consider_active:
+            return internal_patient
+
+        if internal_patient.get("active", False) is not False:
+            return internal_patient
+
+        params = patient_as_search_params(internal_patient)
+        # Ensure it is active
+        internal_patient["active"] = True
```

Comment:
i don't follow why we'd need to mutate the `internal_patient["active"]` value here?  by the time we reach this statement, we must be in consider_active mode and know "active" != False.

what am I missing?  is this in the chain for the process of reactivation?